doc/topics/topology/syndic.rst

```diff
@@ -4,102 +4,187 @@
 
-In order for the high-level master to return information from minions that are
-below the syndic(s), the CLI requires a short wait time in order to allow the
-syndic(s) to gather responses from their minions. This value is defined in the
-``syndic_wait`` and has a default of five seconds.
+A Minion node must run a ``salt-minion`` daemon.
 
-While it is possible to run a syndic without a minion installed on the same machine,
-it is recommended, for a faster CLI response time, to do so. Without a minion
-installed on the syndic, the timeout value of ``syndic_wait`` increases
-significantly - about three-fold. With a minion installed on the syndic, the CLI
-timeout resides at the value defined in ``syndic_wait``. 
+When a ``salt-master`` daemon issues a command, it will be received by the
+Syndic and Minion nodes directly connected to it.  A Minion node will process
+the command in the way it ordinarily would.  On a Syndic node, the
+``salt-syndic`` daemon will relay the command to the ``salt-master`` daemon
+running on the Syndic node, which then propagates the command to to the Minions
+and Syndics connected to it.
+
+When events and job return data are generated by ``salt-minion`` daemons, they
+are aggregated by the ``salt-master`` daemon they are connected to, which
+``salt-master`` daemon then relays the data back through its ``salt-syndic``
+daemon until the data reaches the Master or Syndic node that issued the command.
+
+Syndic wait
+===========
 
 .. note::
 
-    To reduce the amount of time the CLI waits for minions to respond, install a minion
-    on the syndic or tune the value of the ``syndic_wait`` configuration.
+    To reduce the amount of time the CLI waits for Minions to respond, install
+    a Minion on the Syndic or tune the value of the ``syndic_wait``
+    configuration.
+
+While it is possible to run a Syndic without a Minion installed on the same
+system, it is recommended, for a faster CLI response time, to do so.  Without a
+Minion installed on the Syndic node, the timeout value of ``syndic_wait``
+increases significantly - about three-fold. With a Minion installed on the
+Syndic, the CLI timeout resides at the value defined in ``syndic_wait``.
+
+.. note::
+
+    If you have a very large infrastructure or many layers of Syndics, you may
+    find that the CLI doesn't wait long enough for the Syndics to return their
+    events.  If you think this is the case, you can set the
+    :conf_master:`syndic_wait` value in the Master configs on the Master or
+    Syndic nodes from which commands are executed.  The default value is ``5``,
+    and should work for the majority of deployments.
+
+In order for a Master or Syndic node to return information from Minions that
+are below their Syndics, the CLI requires a short wait time in order to allow
+the Syndics to gather responses from their Minions. This value is defined in
+the :conf_master:`syndic_wait` config option and has a default of five seconds.
```

Comment:
This section is not quite right. The syndic isn't gathering minion responses during this time, but rather it is using the target string to generate a list of minions it expects to return for the command. (The master does this as well, but doesn't know about minions below the syndic). The `syndic_wait` is the amount of time we wait for the syndic to return this data, before assuming the syndic is dead, or is not going to return this list. If the syndic doesn't return this list of expected minions in time, then the master could assume there are no minions expected to return from below that syndic, and return on the CLI early.

Keep me posted if that doesn't make sense.